recipe/build.sh

```diff
@@ -11,7 +11,7 @@ fi
 if [ $(uname) == Linux ]; then
     XWIN_ARGS="--enable-xcb-shm"
 fi
-LDFLAGS=$(echo $LDFLAGS | sed -e 's/-Wl,--gc-sections//')
+export ax_cv_c_float_words_bigendian="no"
```

Comment:
This should really be restricted to only x86_64bit.

From wikipedia:

> The ARM architecture was little-endian before version 3 when it became bi-endian. 

> The Intel x86 and also AMD64 / x86-64 series of processors use the little-endian format, and for this reason, it is also known in the industry as the "Intel convention".

https://en.wikipedia.org/wiki/Endianness

There was a way to check for the conda architecture in the build.sh script